vignettes/manual/BuildHector.Rmd

```diff
@@ -156,6 +128,55 @@ sure to unset `CXXPROF`, clean the build directories with `make clean`,
 and rebuild when you are ready to go back to
 production runs.
 
-## iOS/Android
 
-Not yet. ;)
+Run Hector from the terminal with the command line. 
+
+```
+./src/hector ./inst/input/name-of-ini.ini
+```
+
+
+
+## Xcode (Mac OSX)
+
+These directions assume a basic familiarity with Xcode and Mac OS X software installation.
+
+* Install [Xcode](https://developer.apple.com/xcode/downloads/) if necessary. Hector has been built and tested with Mac OS >= 10.8.5. The project files are for Xcode 11.3.1.
+* Download and install Boost, following instructions above.
+* Download the [Hector zip file](https://github.com/JGCRI/hector/archive/master.zip) or check out the repository using Git.
+* From Xcode, open the project file in `project_files/Xcode/hector.cxodeproj`.
+* Build the project. See below if you encounter errors.
+* Change the current Scheme settings (Scheme->Edit Scheme) and add a command-line argument (*Arguments* tab, e.g. "input/hector_rcp45.ini").
+* Run!
+
+Xcode Build Settings to check if you encounter build errors, they should match the version installed on your machine:
+
+* *Architectures-Base SDK*: OS X 10.8 [or OS X version on machine] 
+* *Build Options-Compiler for C/C++/Objective-C*: Default compiler (Apple LLVM 5.1)
+* *Search Paths-Header Search Paths*: "/usr/local/include /usr/local/lib/boost_1_52_0/" [or other header directories]
+* *Search Paths-Library Search Paths*: "/usr/local/lib/" [or other library directories]
+* *LLVM-Language-C++-C++ Standard Library*: libc++ (LLVM C++ standard library with C++11 support)
+* *Linking-Other Linker Flags*: "-lgsl -lgslcblas -lm"
```

Comment:
We shouldn't need GSL flags?

Reply:
👍 make sense I suspect that some of these might be out of date

Reply:
`boost_system` and `boost_filesystem` should be the only libraries we need to link against.

Reply:
I am not sure what this comment means, does that mean we can delete the following lines? 

```
* *Search Paths-Library Search Paths*: "/usr/local/lib/" [or other library directories]
* *LLVM-Language-C++-C++ Standard Library*: libc++ (LLVM C++ standard library with C++11 support)
* *Linking-Other Linker Flags*: "-lgsl -lgslcblas -lm"
```

Reply:
We don't need the two gsl flags at the end . The others we need.